mozilla/fx-private-relay
# Block Promotional Emails UI

# Summary

Blocked until https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/61 and https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/63 are merged

This PR lets users set their email forwards prefs to "Block Promotional Emails". ~This PR depends on/is blocked by PR #1530~

TODO: 

- [x] Add translated strings (PRs below should be merged)
  - [x] https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/61
  - [x] https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/63
- [x] Confirm toggle bottle design for free-users (including label setting) 
- [x] Remove "Forward all" label
- [x] Add logic to send correct GA event
- [x] Update label text per previous function (Label for blue/gray hover) 
- [x] Add filter option
- [x] Remove console.logs
- [x] Mobile Breakpoint
- [x] Resolve Linework behind radio buttons

# Screenshot (if applicable)

<img width="1180" alt="image" src="https://user-images.githubusercontent.com/2692333/154768029-cb0264ea-c533-4f51-90f1-1638444c92b3.png">

<img width="1171" alt="image" src="https://user-images.githubusercontent.com/2692333/154768032-3ac2a499-41d5-4667-b64c-8fade833c016.png">

<img width="1162" alt="image" src="https://user-images.githubusercontent.com/2692333/154768036-9cfcb381-fd10-43a0-83eb-d39a73a51e60.png">


# How to test

- Checkout the strings PR: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/63
- From a free account:
  - You should be able to toggle aliases on and off as expected. There should be no visible changes for free users.
- From a premium account:
  - TBD

# Checklist
